src/components/bs5/sidenav/sidenav.scss

```diff
@@ -80,7 +92,20 @@
             text-underline-offset: 0.25rem;
             text-decoration-thickness: 2px;
         }
+    }
+
+    a.nav-link {
```

Comment:
Had to include a.nav-link as the specificity from qld-type was overriding it. Couldn't nest into above scss as it also applied the other styles mentioned there, so pulled the visited styles out one level.